### DISCUSSION TITLE
Need help understanding on how or if we can modify configuration

### DISCUSSION THREAD

**Discussion by adornala** [zwave-js/zwave-js#5636] [General] [answered]
Hello, sorry I am a rookie, I am trying to understand how or if I can update the configuration for Honeywell Home ProA7Plus. 

Why this question?
Recently I bought a Honeywell Home ProA7Plus, I tried to integrate it with Home Assistant, there were a few issues but it worked. I can see the device in HA. When I tried to use HA for arming and disarming the ProA7Pl, I see there are no parameters I could use. I did some research to understand why I couldn't see any options, I eventually found out there are no configuration properties exported in XML [here](https://products.z-wavealliance.org/products/3675?selectedFrequencyId=2#:~:text=Download%20Product%20Data%20in%20XML%20Format%3A%20Download).

Alliance Url - https://products.z-wavealliance.org/products/3675?selectedFrequencyId=2

So was wondering if we can export those and enable options to arm, disarm or get device status updates. I would like to help devs and learn along the line.

Thank you.

**Comment by AlCalzone** [comment, marked as answer]
I don't know of anything in the Z-Wave standard that corresponds to remotely arming an alarm device. If the device does support it, it's probably done via configuration parameters, because the manufacturer is free to choose those.

In any case, you'll have to ask the manufacturer how the functionality you are missing is exposed via Z-Wave. For configuration parameters, we need the documentation because we can't simply add those, without knowing what exactly to add.
